portmaster/install/linux.md

```diff
@@ -179,11 +177,31 @@ WantedBy=multi-user.target
 
 Finally, reload the systemd daemon and enable/start the Portmaster:
 
-```
+```sh
 sudo systemctl daemon-reload
 sudo systemctl enable --now portmaster
 ```
 
+__6bis.__ Start it on boot Runit (systems)
```

Comment:
I'm not too sure about this line. **6bis.** doesn't seem like a step number.

@dhaavi Any idea what should it be?  
This step is only needed for those that step **6.** isn't applicable, and vice-versa.

(i.e. those that use `systemd` init shall need to follow the current step **6.** and skip this step and go to step **7.**)  
(i.e. those that use `runit` init shall need to ignore the current step **6.** and follow this step instead)

Reply:
Maybe we'll could just remove it, keep the `6. ` one and use an unordered list with an item for each init system. Also there is package for the openRC unit system that were could had (I never used it though).

Reply:
> Also there is package for the openRC unit system that were could had (I never used it though).

Exactly. There's runit, openRC, s6, 66, _etc_ _etc_ countless init systems. But among those, `systemd` is the most popular. That's why most projects simply give instructions for systemd.

If you'd ask my honest opinion, I'd say that we should not include the runit section in this doc. (Even though that's currently what I am using)

IMO, most runit users probably understand their init system enough that they can create this same `run` file themselves. 
 
And those people who are using neither systemd nor runit... well they probably understand their init system *completely*.

Reply:
The solution we came up with is to add a new last section to the page:

```
### Community Contributions
```

This will then have it's first sub-section:

```
#### Start Portmaster Automatically with Runit
```
(or similar)

The content from this PR would then go into there.
At the start of point 6 of the manual install guide, you can create a list with anchor links to community contributions:

```
__6.__ Start it on boot

In case you are not using `systemd` as your init system - you most likely know if that is the case - these guides contributed by the community will get you started:
- [Runit](#start-portmaster-automatically-with-runit)
```

Reply:
> The solution we came up with is to add a new last section to the page:
> 
> ```
> ### Community Contributions
> ```
> 
> This will then have it's first sub-section:
> 
> ```
> #### Start Portmaster Automatically with Runit
> ```
> 
> (or similar)
> 
> The content from this PR would then go into there. At the start of point 6 of the manual install guide, you can create a list with anchor links to community contributions:
> 
> ```
> __6.__ Start it on boot
> 
> In case you are not using `systemd` as your init system - you most likely know if that is the case - these guides contributed by the community will get you started:
> - [Runit](#start-portmaster-automatically-with-runit)
> ```

Then that's what Im gonna do.